software/o_c_REV/APP_HEMISPHERE.ino

```diff
@@ -330,11 +338,15 @@ private:
 SETTINGS_DECLARE(HemisphereManager, HEMISPHERE_SETTING_LAST) {
     {0, 0, 255, "Applet ID L", NULL, settings::STORAGE_TYPE_U8},
     {0, 0, 255, "Applet ID R", NULL, settings::STORAGE_TYPE_U8},
-    {0, 0, 65535, "Data L low", NULL, settings::STORAGE_TYPE_U16},
-    {0, 0, 65535, "Data R low", NULL, settings::STORAGE_TYPE_U16},
-    {0, 0, 65535, "Data L high", NULL, settings::STORAGE_TYPE_U16},
-    {0, 0, 65535, "Data R high", NULL, settings::STORAGE_TYPE_U16},
-    {0, 0, 65535, "Clock data", NULL, settings::STORAGE_TYPE_U16},
+    {0, 0, 65535, "Data L byte 1", NULL, settings::STORAGE_TYPE_U16},
+    {0, 0, 65535, "Data R byte 1", NULL, settings::STORAGE_TYPE_U16},
+    {0, 0, 65535, "Data L byte 2", NULL, settings::STORAGE_TYPE_U16},
+    {0, 0, 65535, "Data R byte 2", NULL, settings::STORAGE_TYPE_U16},
```

Comment:
consider STORAGE_TYPE_U32 during cleanup

Reply:
I'm leaning away from this because of the way the data is stored as alternating 16-bit values for each hemisphere means that would turn into garbage if each half now was using 32-bit values (since it would get loaded with both hemisphere's data as a single hemisphere's data)